plugins/woocommerce/tests/e2e-pw/tests/merchant/create-cart-block.spec.js

```diff
@@ -23,6 +23,14 @@ test.describe(
 		} ) => {
 			await goToPageEditor( { page } );
 
+			// Dismiss "Choose a pattern" if present (Pressable)
+			// eslint-disable-next-line playwright/no-conditional-in-test
+			if (
+				await page.getByLabel( 'Close', { exact: true } ).isVisible()
+			) {
+				await page.getByLabel( 'Close', { exact: true } ).click();
+			}
+
```

Comment:
On Pressable, the Choose pattern pops up every time. make sure to dismiss it before proceeding.

Reply:
Updated this with `closeChoosePatternModal`